src/clr-core/styles/variables/_variables.global.scss

```diff
@@ -44,7 +44,7 @@ $clr_baselineRem_10: convertBaselineToBase20(240); // 240px
 $clr_baselineRem_15: convertBaselineToBase20(360); // 360px
 
 // Pixel-based utilities
-$clr_baselineRem_-1px: -convertBaselineToBase20(1);
+$clr_baselineRem_-1px: -1px;
```

Comment:
We can't do this. It would break browser scaling. If you need a -1px measurement, your best bet is to use...

`$clr_baselineRem_-1px: -1 * $clr_baselineRem_1px`

...or you could use `-1 * $clr_baselineRem_1px` in your code to prevent us from starting a precedent of creating variables for negative values. That could potentially double the number of variables in our SASS...

Reply:
Ended up not needing this so reverted the change.